src/xdocs/beginning_development.xml

```diff
@@ -31,6 +31,14 @@
             git clone git@github.com:you_user_name/checkstyle.git
         </source>
       </div>
+      <p>
+        4. Before opening project in IDEA do build in terminal by running in repository root folder
+      </p>
+      <div class="wrap-content">
+        <source>
+            mvn install
```

Comment:
this will require contributor to have maven installed (see item 1)

Reply:
mvn is added.
![image](https://user-images.githubusercontent.com/812984/147105928-35504c00-d288-409e-b2db-138819f67a49.png)

Reply:
maven 3.0.1 works with any JDK version starting from 7. But there is some issues.
The recommended version of maven is `3.8.x` for JDK17 and  `3.6.x` for JDK 11

Reply:
I used version from our pom